Sources/Extensions/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -46,3 +46,20 @@ public extension Optional {
 	}
 	
```

Comment:
Hey @buddax2 , Just as a suggestion I think will be better put this extension as an Optional extension instead of global generic T. I mean, put this as an operator of Optional. Something like:
```swift 
// MARK: - Operators
infix operator ??= : AssignmentPrecedence

extension Optional {

    public static func ??= (lhs: inout Optional, rhs: Optional) {
        guard let rhs = rhs else { return }
        lhs = rhs
    }
}

```
I think it will provide the same behavior and makes more sense since the operator is applied to optional values only :)

Reply:
Good thoughts 👍

Reply:
@LucianoPAlmeida great. Done

Reply:
Another thing, have we thought about using `||=` for the operator?

Reply:
@SD10 for me personally `??=` looks more like optional rather than `||=`.

Reply:
@buddax2 I was only suggesting this because I think other languages use the `||=` operator for this functionality. Thus, it would be more familiar. When I saw `??=` I had no idea what it did until I looked at the code

Reply:
@SD10 if I'm not mistaken `||=` works slightly in a different way for example in Ruby. It assigns new value only if there was no value in the var before. So `a ||= b` is more like `a || a = b`. 

But here in our case we want to assign `b` only in case if `b != nil`

Reply:
@buddax2 I see what you mean, thanks for clarifying 👍

Reply:
@buddax2 Thank's for the changes :)